lib/openstudio-standards/standards/Standards.AirLoopHVAC.rb

```diff
@@ -3414,4 +3409,42 @@ def air_loop_hvac_multi_stage_dx_cooling?(air_loop_hvac)
 
     return dx_clg
   end
+
+  # Create an economizer maximum OA fraction schedule with
+  # For ASHRAE 90.1 2019, a maximum of 75% to reflect damper leakage per PNNL
+  #
+  # @param air_loop_hvac [OpenStudio::Model::AirLoopHVAC] HVAC air loop object
+  # @param oa_control [OpenStudio::Model::ControllerOutdoorAir] Outdoor air controller object to have this maximum OA fraction schedule
+  # @param snc [String] System name
+  #
+  # @return [OpenStudio::Model::ScheduleRuleset] Generated maximum outdoor air fraction schedule for later use
+  def set_max_frac_oa_sch(air_loop_hvac, oa_control, snc)
```

Comment:
set_maximum_fraction_outdoor_air_schedule

Reply:
Addressed in last commit.